frontend/public/components/routes.tsx

```diff
@@ -102,15 +100,31 @@ export const RouteStatus: React.SFC<RouteHostnameProps> = ({obj: route}) => {
     }
   });
 
-  return atLeastOneAdmitted
-    ? <span>
-      <div className="fa fa-check text-success co-m-route-status-icon" aria-hidden="true"></div>
+  return atLeastOneAdmitted ? 'Accepted' : 'Rejected';
+};
+
+export const RouteStatus: React.SFC<RouteHostnameProps> = ({obj: route}) => {
```

Comment:
We shouldn't reuse `RouteHostnameProps` here. Create a `RouteStatusProps`.

Reply:
For my understanding, why not reuse a single `type` instead of making multiple types with the same form (I'm inclined to do similar in what I'm currently working on)?  Similar to why we ` export type abc` when we really only use them within a file.  I'm just curious about the convention, it seems to be more verbose than necessary.

Reply:
thanks, fixed

Reply:
> For my understanding, why not reuse a single type instead of making multiple types with the same form

It seems very specific to me: you're saying this particular component takes these props. If you ever add another prop later, you'd have to separate the types anyway (and might accidentally add a prop to another component if you don't realize it's being reused).

Having said that, I don't know what the best practice is. If we do share, I would give it a generic name. I don't think it's worth it here to save 3 lines of code.

> Similar to why we export type abc when we really only use them within a file

That we shouldn't do :)

Reply:
Cool, works for me.